frontend/src/pages/analytics/explore/AnalyticsExplore.tsx

```diff
@@ -182,9 +182,10 @@ const AnalyticsExplore: React.FC = () => {
     } else if (!pipeline) {
       setRenderCondition(ExploreRenderCondition.Empty);
     } else if (
-      !SolutionVersion.Of(pipeline.templateInfo?.pipelineVersion ?? '').equalTo(
-        SolutionVersion.Of(pipeline.templateInfo?.solutionVersion ?? '')
-      )
+      SolutionVersion.Of(pipeline.templateInfo?.pipelineVersion ?? '')
```

Comment:
We should only disable the exploration for the pipeline version earilier than 1.1.6.

Should we use `SolutionVersion.Of(pipeline.templateInfo?.pipelineVersion ?? '').lessThan(SolutionVersion.V_1_1_6)`?

Reply:
No, explorations enable when pipeline short version equal to control plane short version. Because the reporting generated SQL only supports the latest schema.

Reply:
We introduced breaking changes to data schema since 1.1.6. However, the data schema of the latest (1.2.0) is compatible with 1.1.6. Why do we disable exploration for pipeline 1.1.6 in the latest console?

Reply:
Got it!